.github/workflows/main.yml

```diff
@@ -56,29 +54,39 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        cversion: ['4.11.0', '4.14.0', '23.7.3', '23.5.2', '23.3.1', '23.1.0', '22.11.1']
+        cversion: ['4.11.0', '4.14.0', '22.11.1', '23.3.1', '23.7.3', '23.10.0', '24.1.2']

```

Comment:
That reminds me that we should maybe consider running the tests with the canary builds we create for conda on merging into main (but don't fail), I opened #62